docs/design-docs/01-explicit-only-namespace-strategy.md

```diff
@@ -0,0 +1,101 @@
+# Explicit-only Namespace Strategy for `RootSync`s
+
+* Author(s): @tomasaschan
+* Approver: TBD
```

Comment:
LGTM

There was some discussion about potentially implementing this as a spec field rather than an annotation. Assigning @janetkuo as approver so she can weigh in and approve the API decision.

/assign @janetkuo

Reply:
/lgtm